hw8/decorator.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+
+def decorator(func):
+    def wrapper(*args, **kwargs):
+        t = datetime.now()
+        print(f'====\nFunction call time - {t.strftime("%H:%M:%S")}\n====')
+        func(*args, **kwargs)
```

Comment:
You should call your function inside of the wrapper only once. Here you call it in line 8 and in line 10. Remove line 8 to make it work correctly